relay.go

```diff
@@ -134,15 +209,15 @@ func (r *Relayer) handleCallReq(f lazyCallReq) error {
 			ErrField(err),
 			LogField{"hostPort", hostPort},
 		).Warn("Failed to connect to relay host.")
-		// TODO : return an error frame.
-		return nil
+		return ErrNoPeers
```

Comment:
discard this change -- this should be a separate error that needs to be retried. i'll do a separate diff for that